_qs/410-exercise-service.md

```diff
@@ -41,11 +41,11 @@ In general, a provider bundle should export the API it _provides_; in our case w
 
 ### Change the User Application
```

Comment:
It says User Application in the heading here. I think it is supposed to be User Application that uses the Upper _service_.